examples/logging/local_debug.md

```diff
@@ -0,0 +1,43 @@
+This logging configuration is great for running Nautbot locally in docker with `DEBUG=True`.  It is 
+impractical to use in production due to the multi-line output:
+
+```python
+LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "normal": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s :\n  %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+        "verbose": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)-20s %(filename)-15s %(funcName)30s() :\n  %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+    },
+    "handlers": {
+        "normal_console": {
+            "level": "INFO",
+            "class": "rq.utils.ColorizingStreamHandler",
+            "formatter": "normal",
+        },
+        "verbose_console": {
+            "level": "DEBUG",
+            "class": "rq.utils.ColorizingStreamHandler",
```

Comment:
As you mentioned in Slack, we should probably move away from `rq`-specific logging handler classes.

Reply:
@glennmatthews I went ahead and removed examples as well as docs and config utilizing `rq.utils.ColorizingStreamHandler`